### PR TITLE
Adding code for histogram save graph

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons in this save graph code, i have some querries that i would like to get some clarification.
HAve a look and let me know what you think about this

File: instat/dlgBoxPlot.vb [hunk 1]
```diff
@@ -134,4 +134,13 @@ Public Class dlgBoxplot
...
     End Sub
+    Private Sub ucrSaveBoxplot_GraphNameChanged() Handles ucrSaveBoxplot.GraphNameChanged, ucrSaveBoxplot.SaveGraphCheckedChanged
+        If ucrSaveBoxplot.bSaveGraph Then
+            ucrBase.clsRsyntax.SetAssignTo(ucrSaveBoxplot.strGraphName, strTempDataframe:=ucrSelectorBoxPlot.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempGraph:=ucrSaveBoxplot.strGraphName)
+            ucrBase.clsRsyntax.bExcludeAssignedFunctionOutput = True
+        Else
+            ucrBase.clsRsyntax.SetAssignTo("last_graph", strTempDataframe:=ucrSelectorBoxPlot.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempGraph:="last_graph")
+            ucrBase.clsRsyntax.bExcludeAssignedFunctionOutput = False
+        End If
+    End Sub
```
Comment (dannyparsons) [inline instat/dlgBoxPlot.vb]: You've also edited this your other pull request. Isn't that going to cause conflicts?

**Comment by maxwellfundi** [comment]
Am going to try delete that commit to avoid conflict

**Comment by maxwellfundi** [comment]
@dannyparsons 
Am closing this because i am putting up a corrected version of this pull request.
Thanks

**Pull request closed without merging**